src/backend/metal/src/command.rs

```diff
@@ -1218,6 +1219,7 @@ impl CommandSink {
     }
 
     /// Switch the active encoder to render by starting a render pass.
+    /// Second returned value is `true` if the switch has just happened.
```

Comment:
Looks like a leftover comment? I don't see a second return value here

Reply:
oh right, let me fix that!